pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools>=61.0.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
-packages = ["xl2times"]
+packages = ["xl2times", "utils"]
```

Comment:
Would this also install a `utils` package to the python (virtual)env if someone does `pip install .` in the root directory? Wondering if that might result in name clashes. 

I was thinking of `utils` more as a collection of scripts for benchmarking/developing, and not as a package to be distributed. Do we need to include it in `pyproject.toml`? Happy to go with the best practice here.

Reply:
Ah apologies, this was a fix for a ModuleNotFound from a `from utils.dd_to_csv` import in run_benchmarks I added, but the right solution was to `import dd_to_csv` directly.
I'll revert this change in #186 

But yes, this would have included `utils` in built wheels etc. I didn't realise this was meant to not be included.